optoforce/launch/optoforce_hand.launch

```diff
@@ -0,0 +1,12 @@
+<launch>
+  <arg name="rviz" default="true"/>
+
+  <include file="$(find optoforce)/launch/optoforce.launch"/>
+
+  <include if="$(arg rviz)" file="$(find optoforce)/launch/rviz.launch"/>
+
+  <!-- Static transforms form Optoforce sensor to finger tip -->
+  <node name="optoforce_1_broadcaster" pkg="tf" type="static_transform_publisher" args="0 0 0 0 0 1.57 rh_rftip optoforce_1 100" />
```

Comment:
a shame that we have to manually specify the finger tips (what happens when we put them on a full hand). Report an issue for later?